sonic_platform_base/chassis_base.py

```diff
@@ -197,6 +232,19 @@ def get_module(self, index):
 
         return module
 
+    def get_module_index(self, module_name):
+        """
+        Retrieves module index from the module name
+
+        Args:
+            name: A string, prefixed by SUPERVISOR, LINE-CARD or FABRIC-CARD
```

Comment:
Arg is called `name` here, but in the method signature it is `module_name`. Please align them.

Reply:
Fixed.